scss/_patterns_side-navigation-expandable.scss

```diff
@@ -28,15 +29,54 @@
 
     &::before {
       @extend %icon;
-      @include vf-icon-chevron;
+      @include vf-icon-chevron($colors--light-theme--text-inactive);
       content: '';
       transform: rotate(-90deg);
       transition: transform 100ms;
     }
+
+    &:hover {
+      background: $colors--theme--background-hover;
+      color: $colors--theme--text-default;
+    }
+  }
+
+  // FIXME: Temporary workaround for unsupported theme colors in vf-icon-chevron
+  .p-side-navigation__expand.is-light::before {
+    @include vf-icon-chevron($colors--light-theme--text-inactive);
```

Comment:
As these 4 overrides are the same (light theme color icon) would be good to group them to avoid duplication. Also, use `body.is-light` so that we limit the support to main page theme, because making it on any parent will be unreliable anyway.

So something like:

```scss
.p-side-navigation__expand.is-light::before,
body.is-light .p-side-navigation__expand::before,
.p-side-navigation__expand.is-paper::before,
body.is-paper .p-side-navigation__expand::before {
  @include vf-icon-chevron($colors--light-theme--text-inactive);
}
```